monkey/monkey_island/cc/ui/src/components/report-components/ransomware/AttackSection.tsx

```diff
@@ -0,0 +1,102 @@
+import React, {ReactElement, ReactFragment, useEffect, useState} from 'react';
+import IslandHttpClient from '../../IslandHttpClient';
+import {FileEncryptionTable, TableRow} from './FileEncryptionTable';
+import NumberedReportSection from './NumberedReportSection';
+import LoadingIcon from '../../ui-components/LoadingIcon';
+
+const ATTACK_DESCRIPTION = 'After the attacker or malware has propagated through your network, \
+                            your data is at risk on any machine the attacker can access. It can be \
+                            encrypted and held for ransomware, exfiltrated, or manipulated in \
+                            whatever way the attacker chooses.'
+const HOSTNAME_REGEX = /^(.* - )?(\S+) :.*$/;
+
+function AttackSection(): ReactElement {
+  const [tableData, setTableData] = useState(null);
+
+  useEffect(() => {
+    IslandHttpClient.get('/api/telemetry?telem_category=file_encryption')
+      .then(resp => setTableData(processTelemetry(resp.body)));
+  }, []);
+
+
+  if (tableData == null) {
+      return <LoadingIcon />
+  }
+
+  return (
+    <NumberedReportSection
+      index={3}
+      title='Attack'
+      description={ATTACK_DESCRIPTION}
+      body={getBody(tableData)}
+    />
+  );
```

Comment:
What happens when there's no encryptions? We show empty table? I'd prefer we render a message like "No files in the network got encrypted. See how to [setup encryption simulation](https://www.guardicore.com/infectionmonkey/docs/usage/use-cases/)"

Reply:
![image](https://user-images.githubusercontent.com/19957806/127002234-82c0001f-efb6-4bc0-89a4-6dd83e3f7df4.png)